wallet/src/main/resources/application.properties

```diff
@@ -1,15 +1,19 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/wallet?characterEncoding=utf-8&useSSL=false&allowPublicKeyRetrieval=true&createDatabaseIfNotExist=true
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=redmarli43
```

Comment:
I this is your "shared" pass - I hope you will change it because you shared it with the internet

Reply:
@aleksandrbeljakov I totally agree with you, I messed a little bit up. But I never use a such small and easy password, and of course, I enable mfa if it's possible :)